client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -94,8 +96,9 @@ export default function Dashboard(): JSX.Element {
           <CreateColumnDialog createColumnOpen={createColumnOpen} handleCloseCreateColumn={handleCloseCreateColumn} />
         </Grid>
 
-        <Grid item>
-          <BoardUI />
+        <Grid item xs={12}>
+          <BoardUI boardData={initialData} />
```

Comment:
I believe it's a better idea for the Board itself to load the board data, mock or otherwise.